app/views/shared/_field_errors.html.erb

```diff
@@ -5,9 +5,13 @@
   <div class="govuk-error-summary__body">
     <ul class="govuk-list govuk-error-summary__list">
       <% errors.messages.reject { |_f, m| m.empty? }.each do |field, _messages| %>
-        <li>
-          <%= link_to errors.full_messages_for(field).first, "##{form.object.class.name.underscore}_#{field}_error" %>
-        </li>
+        <% if local_assigns[:disable_refresh] %>
+          <li>
+            <%= link_to errors.full_messages_for(field).first, "##{field}", data: { turbolinks: false } %>
+          </li>
+        <% else %>
+          <li><%= link_to errors.full_messages_for(field).first, "##{field}" %></li>
+        <% end %>
```

Comment:
I'm not sure I understand why turbolinks is causing problems here, but I wonder if it is possible to replace the two mostly duplicated lines with a single line.  This is currently the equivalent of 
```ruby
if true 
   return false 
else 
   return true 
end 
``` 
and it should be possible to reverse the logic so :disable_refresh becomes :enable_refresh and then the single line can be ...  
```
<%= link_to errors.full_messages_for(field).first, "##{field}", data: { turbolinks: local_assigns.fetch(:enable_refresh, false) } %>
``` 
or something similar.